app/controllers/salt_controller.rb

```diff
@@ -17,7 +17,7 @@ def accept_minion
 
     respond_to do |format|
       format.html { redirect_to root_path }
-      format.json { head :ok }
+      format.json { render nothing: true, status: :accepted }
```

Comment:
I think `:ok` resembles better what happens here in reality than `accepted`. `accepted` has the connotation of having added the task to a job queue. In this case, the accept process happens synchronously, and we are actually waiting for the `salt-api` to return to us.

Reply:
I thought that it would be fine to use since we delegate the task to salt-api and return always a success response. I'll revert the change since it won't break the fix anyway.

Reply:
Done.